templates/project/.github/workflows/test.yaml.twig

```diff
@@ -76,9 +76,12 @@ jobs:
             - name: Add PHPUnit matcher
               {% verbatim %}run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"{% endverbatim %}
 
+{# Remove when all branches supports php 8.0 #}
+{% if '8.0' != branch.highestPhpVersion.toString %}
```

Comment:
Not sure if this is ok about blank lines, I always add or remove one 😞

Reply:
Lets see after merge 😄 

Please keep an eye on the dev kit PRs, thanks!